Automattic/newspack-e2e-tests
# 3. Add theme snapshots

This PR adds visual regression snapshots for themes: all 5 of them, homepage and single post page for each, both on desktop and a phone. 

Also changes to run the E2E tests using a public-facing domain are included here for testing plugins which require one. But testing that in itself is a to do. 

## How to test

1. Create a `scripts/.env` file based on `scripts/.env-sample` file. This sets values for ngrok and the WP version to test against. Ngrok values can be omitted.
1. Spin up the environment according to the README
1. Run visual regression tests: `$ npm run test:visual:check`. These should complete without errors.
